app/bot/llm.py

```diff
@@ -28,10 +28,13 @@ async def generate_response(
             try:
                 async with cs.post(url, json={
                     "model": _botconf.botconfig.llm_model,
-                    "prompt": message.content,
-                    "system": system_prompt,
-                    "context": [],
                     "stream": False,
+                    "messages": [
+                        {"role": "system",
+                        "content": system_prompt},
+                        {"role": "user",
+                        "content": message.content},
+                    ],
```

Comment:
Still doesn't have history over time, but this does get you closer to migrating and should be functionally equivalent.

Reply:
Yes, this is a quick change to just migrate to the chat API